src/Symfony/Bridge/Twig/Tests/AppVariableTest.php

```diff
@@ -258,7 +253,7 @@ protected function setTokenStorage($user)
         $token->method('getUser')->willReturn($user);
     }
 
-    private function setFlashMessages()
+    private function setFlashMessages($startSession = true)
```

Comment:
I think you should rename to `sessionHasStarted`

Reply:
Done